applications/nrf5340_audio/src/audio/sw_codec_select.c

```diff
@@ -170,9 +174,11 @@ int sw_codec_encode(void *pcm_data, size_t pcm_size, uint8_t **encoded_data, siz
 			return -ENODEV;
 		}
 
-		*encoded_data = m_encoded_data;
-		*encoded_size = encoded_bytes_written;
+		audio_frame->meta.data_coding = LC3;
```

Comment:
Something that caught my eye here is the naming: 
You store the value LC3 in the meta.data_coding which can be a bit confusing when comparing the the metadata of LE Audio (where the codec information is specifically _not_ in the metadata). 

Not something that needs to be changed, but something to consider at least :)

Reply:
This audio struct is for the internal audio modules and not for the LE Audio, the two should be viewed separately.